Source/MQTTnet/Implementations/CrossPlatformSocket.cs

```diff
@@ -86,6 +86,7 @@ public void Bind(EndPoint localEndPoint)
         {
             if (localEndPoint is null) throw new ArgumentNullException(nameof(localEndPoint));
 
+            _socket.SetSocketOption(SocketOptionLevel.Socket, SocketOptionName.ReuseAddress, true);
```

Comment:
I don't think we need this. There is already a property which sets the socket option. The property is also set when the server starts. Probably the user who has problems with that only needs to enable this in the server options.

Reply:
Ok, can you tell him how to enable this, please? https://github.com/chkr1011/MQTTnet/issues/872.

I will close the pull request soon.

Reply:
Will be closed as I have given a hint to the original person who had this problem.